tests/func/test_fs.py

```diff
@@ -468,6 +469,7 @@ def test_download_callback(tmp_dir, dvc, cloud, local_cloud):
     fs = cls(**config)
 
     (tmp_dir / "to_upload").write_text("foo")
+    cloud.mkdir(exist_ok=True, parents=True)
```

Comment:
If we need for ssh, this should already be created in `ssh` fixture. Note  that `oss` fixture does not implement anything, it's mostly a wrapper for config right now, so it fails on `mkdir`.